scripts/bad-linebreaks.mjs

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+
+import fs from 'fs';
+import { glob } from 'glob';
+
+function getLine(txt, index) {
+
+  let line = 1;
+
+  for (let i = 0; i < index; i++) {
+    if (txt[i] === '\n') {
+      line += 1;
+    }
+  }
+
+  return line;
+
```

Comment:
```suggestion
    return txt.slice(0, index).replace(/[^\n]/g, '').length + 1;
```
would this work?

Reply:
> would this work?

Aren't string operations costlier than simple traversal?

Reply:
i'm not sure why performance would literally ever matter here

Reply:
this is a lot slower, which is why I favored the unga bunga loop

Reply:
in theory sure, but how impactful is it in practice?